test1/task3/main.c

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+void outputComments(char* filename)
```

Comment:
```suggestion
void outputComments(const char* filename)
```
Мы вряд ли будем имя файла менять